project_testing/housing_characteristics/County.tsv

```diff
@@ -1,2 +1,3 @@
y	Option=WI, Lincoln County	Option=WI, Manitowoc County	Option=WI, Marathon County	Option=WI, Marinette County	Option=WI, Marquette County	Option=WI, Menominee County	Option=WI, Milwaukee County	Option=WI, Monroe County	Option=WI, Oconto County	Option=WI, Oneida County	Option=WI, Outagamie County	Option=WI, Ozaukee County	Option=WI, Pepin County	Option=WI, Pierce County	Option=WI, Polk County	Option=WI, Portage County	Option=WI, Price County	Option=WI, Racine County	Option=WI, Richland County	Option=WI, Rock County	Option=WI, Rusk County	Option=WI, Sauk County	Option=WI, Sawyer County	Option=WI, Shawano County	Option=WI, Sheboygan County	Option=WI, St. Croix County	Option=WI, Taylor County	Option=WI, Trempealeau County	Option=WI, Vernon County	Option=WI, Vilas County	Option=WI, Walworth County	Option=WI, Washburn County	Option=WI, Washington County	Option=WI, Waukesha County	Option=WI, Waupaca County	Option=WI, Waushara County	Option=WI, Winnebago County	Option=WI, Wood County	Option=WV, Barbour County	Option=WV, Berkeley County	Option=WV, Boone County	Option=WV, Braxton County	Option=WV, Brooke County	Option=WV, Cabell County	Option=WV, Calhoun County	Option=WV, Clay County	Option=WV, Doddridge County	Option=WV, Fayette County	Option=WV, Gilmer County	Option=WV, Grant County	Option=WV, Greenbrier County	Option=WV, Hampshire County	Option=WV, Hancock County	Option=WV, Hardy County	Option=WV, Harrison County	Option=WV, Jackson County	Option=WV, Jefferson County	Option=WV, Kanawha County	Option=WV, Lewis County	Option=WV, Lincoln County	Option=WV, Logan County	Option=WV, Marion County	Option=WV, Marshall County	Option=WV, Mason County	Option=WV, McDowell County	Option=WV, Mercer County	Option=WV, Mineral County	Option=WV, Mingo County	Option=WV, Monongalia County	Option=WV, Monroe County	Option=WV, Morgan County	Option=WV, Nicholas County	Option=WV, Ohio County	Option=WV, Pendleton County	Option=WV, Pleasants County	Option=WV, Pocahontas County	Option=WV, Preston County	Option=WV, Putnam County	Option=WV, Raleigh County	Option=WV, Randolph County	Option=WV, Ritchie County	Option=WV, Roane County	Option=WV, Summers County	Option=WV, Taylor County	Option=WV, Tucker County	Option=WV, Tyler County	Option=WV, Upshur County	Option=WV, Wayne County	Option=WV, Webster County	Option=WV, Wetzel County	Option=WV, Wirt County	Option=WV, Wood County	Option=WV, Wyoming County	Option=WY, Albany County	Option=WY, Big Horn County	Option=WY, Campbell County	Option=WY, Carbon County	Option=WY, Converse County	Option=WY, Crook County	Option=WY, Fremont County	Option=WY, Goshen County	Option=WY, Hot Springs County	Option=WY, Johnson County	Option=WY, Laramie County	Option=WY, Lincoln County	Option=WY, Natrona County	Option=WY, Niobrara County	Option=WY, Park County	Option=WY, Platte County	Option=WY, Sheridan County	Option=WY, Sublette County	Option=WY, Sweetwater County	Option=WY, Teton County	Option=WY, Uinta County	Option=WY, Washakie County	Option=WY, Weston County
```

Comment:
I think you should choose up to 6 counties as options. Corresponding epw files will need to be added to `resources/measures/HPXMLtoOpenStudio/weather/project_testing` (which are used in unit testing).

Reply:
Oh. You are right. I think this commit is going to fail the tests.